functions/two_sum.py

```diff
@@ -1,2 +1,5 @@
 def twoSum(n, t):
-    pass
+    for i in range(len(n)):
```

Comment:
this function looks good! perhaps we could add a docstring to describe its functionality.